drivers/core_demographics_report/app/models/core_demographics_report/age_calculations.rb

```diff
@@ -152,25 +174,47 @@ def age_data_for_export(rows)
         rows["_#{age_category_title} Break"] ||= []
         rows["*#{age_category_title}"] ||= []
         rows["*#{age_category_title}"] += ['Gender', nil, 'Count', 'Average Age', nil]
+        available_coc_codes.each do |coc_code|
+          rows["*#{age_category_title}"] += [coc_code]
+        end
+        rows["*#{age_category_title}"] += [nil]
         rows["_#{age_category_title} - All"] ||= []
         rows["_#{age_category_title} - All"] += ['All', nil, send("#{age_category}_count"), send("average_#{age_category}_age"), nil]
+        available_coc_codes.each do |coc_code|
+          rows["_#{age_category_title} - All"] += [age_coc_count(age_category, coc_code.to_sym)]
+        end
```

Comment:
Similarly, I think this could be written like the following, which might be easier to follow, though maybe not.  I'm not as convinced about this one since it still requires iterating through the array.
```suggestion
        rows["_#{age_category_title} - All"] += available_coc_codes.map { |coc_code| [age_coc_count(age_category, coc_code.to_sym)] }

```